crates/ty_ide/src/completion.rs

```diff
@@ -472,6 +472,8 @@ mod tests {
 ",
         );
         test.assert_completions_include("filter");
+        test.assert_completions_do_not_include("_T");
+        test.assert_completions_do_not_include("__annotations__");
```

Comment:
Note I'm guessing we do not want this last assertion.

Reply:
I took the time to do this https://github.com/astral-sh/ruff/pull/19102/commits/fc3b1ed523e319c7d291b0c35aea804cf4472453 as it seems wrong as-is.